README.md

```diff
@@ -79,3 +79,66 @@ The current implementation uses goroutines very liberally which means the query
 ### Plan optimization
 
 The current implementation creates a physical plan directly from the PromQL abstract syntax tree. Plan optimizations not yet implemented and would require having a logical plan as an intermediary step.
+
+
+## Latest benchmarks
+
+Single core benchmarks
+```markdown
+vector_selector/current_engine         	      44	  27756507 ns/op	16146356 B/op	   69121 allocs/op
```

Comment:
Maybe we can use `benchstat` and make these figures more readable both in rendered + unrendered markdown? 🙂

Reply:
Will do !

Reply:
I think it would be nice to run benchmark on each CI to see the stats. We can have an issue to track.

Reply:
That's a good idea 👍

Reply:
Updated to the benchstat format. I also added a make target so we can easily reproduce benchmarks, and potentially hook them into the CI in the future.